include/enmasse-kubernetes.jsonnet

```diff
@@ -10,7 +10,7 @@ local flavorConfig = import "flavor.jsonnet";
     "items": [ templateConfig.generate(use_sasl, with_kafka, false),
                addressController.deployment("enmasseproject/address-controller", "false", 0),
                restapiRoute.ingress(""),
-               addressController.service,
+               addressController.service("LoadBalancer"),
```

Comment:
@ppatierno I'm actually wondering if the appropriate thing to do here is to create additional services that have type=LoadBalanced. The messaging service for instance, contains 2 additional ports that are used for 1. interrouter communication and 2. brokers. I'm pondering whether it is a feature to have them exposed or not.